setup/centos5/resources/jenkins.initd

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+#
+# /etc/init.d/jenkins
+# init script for a jenkins slave.
+#
+# chkconfig: 345 90 60
+# description: jenkins slave
+#
+
+RETVAL=0
+
+NAME=`basename "$0"`
+JENKINS_CONFIG=/etc/sysconfig/$NAME
+LOCK_FILE=/var/lock/$NAME
+JAVA=/usr/bin/java
+PIDFILE=/var/run/jenkins/$NAME.pid
+
+# Source function library.
+. /etc/rc.d/init.d/functions
+
+# Read config
+[ -f "$JENKINS_CONFIG" ] && . "$JENKINS_CONFIG"
+
+JENKINS_PATH=""
+
+# Add SCL path if it exists to avoid using `scl enable`
+[ $SCL ] && JENKINS_PATH=$SCL:$JENKINS_PATH 
+
+# Add bin last since we link ccache in there
+JENKINS_PATH=/home/iojs/bin:$JENKINS_PATHPATH
+
+slave_start() {
+  echo Starting $NAME...
+
+  mkdir `dirname $PIDFILE` > /dev/null 2>&1 || true
+  chown $JENKINS_SLAVE_USER `dirname $PIDFILE`
+
+  ENV="PATH=$JENKINS_PATH NODE_COMMON_PIPE=$NODE_COMMON_PIPE OSTYPE=$OSTYPE"
+  runuser -m -l $JENKINS_SLAVE_USER -c "$ENV $JAVA -jar $JENKINS_SLAVE_JAR $JENKINS_SLAVE_ARGS > $JENKINS_SLAVE_LOG 2>&1 &"
+  secret=`echo $JENKINS_SLAVE_ARGS | egrep -o "\-secret \w+" | cut -d " " -f 2`
```

Comment:
yeah, why don't we just split it up into:
```sh
JENKINS_HOST="jenkins-iojs.nodesource.com"
JENKINS_SLAVE_NAME="iojs-digitalocean-centos5-{{id}}"
JENKINS_SECRET="{{secret}}"
```

and reconstruct `JENKINS_SLAVE_ARGS` from all of that?
+OSTYPE=linux-gnu